src/features/header-flyout/styles/HeaderFlyout.scss

```diff
@@ -9,18 +10,13 @@
 }
 
 .header-flyout-overlay {
-    width: $flyout-list-width;
-    height: $flyout-list-height;
-
     .overlay {
         @include bdl-Overlay-container;
 
+        width: $flyout-list-width;
+        height: $flyout-list-height;
```

Comment:
This is handled by stateful css which is set by the class. By default, the width and height are defined, but the content container is changed depending on if a header or footer is defined

Reply:
Gotcha. What's the reason for moving this here though?

Reply:
It seems to reveal that the math doesn't entirely add up. If we have a both a header and a footer then:

- top padding is 24px
- header/title height is 28px
- container w/ header and footer is 346px
- footer height is 56px

which totals to 454px but it's designed around a height of 460px.

Although it's veryyy minor, moving `height: $flyout-list-height;` to `.overlay` seems to add additional pixels below the footer. The reason it wasn't noticed before is because `height: $flyout-list-height;` on `.header-flyout-overlay` doesn't seem to do anything (or not something obvious).

Reply:
You got a good eye. Thanks for noticing that. You're right. I think I was trying to solve an issue differently but didn't change it back and didn't notice a difference. I moved it back and tested it